modules/alb/main.tf

```diff
@@ -68,19 +91,25 @@ resource "aws_lb_listener" "http" {
 
   default_action {
     type = "redirect"
-
     redirect {
       port        = "443"
       protocol    = "HTTPS"
       status_code = "HTTP_301"
     }
   }
-
 }
 
-// if there are any other distict certificates, add them to the listener
+# if there are any other distinct certificates, add them to the listener
 resource "aws_lb_listener_certificate" "additional_certs" {
   for_each        = var.additional_certificate_arns
   listener_arn    = aws_lb_listener.https_listener.arn
   certificate_arn = each.value
 }
+
+resource "aws_lb_target_group" "main" {
+  name        = "${var.namespace}-${var.stage}-tg"
+  port        = 80
+  protocol    = "HTTP"
+  vpc_id      = var.vpc_id
+  target_type = "instance"
+}
```

Comment:
Is this target group required? It looks like it's not associated with anything looking at the diff

```suggestion
```